lib/presentation/crowd_action/widgets/custom_scrollphysics.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+
+class CustomScrollPhysics extends ScrollPhysics {
```

Comment:
I would avoid calling classes `CustomSomething`.
This is currently only used by the horizontal carousel on the main screen. Wouldn't `CarouselScrollPhyisccs` be a better class name?